src/modules/peek/Peek.FilePreviewer/Previewers/WebBrowserPreviewer/Helpers/MonacoHelper.cs

```diff
@@ -81,13 +81,14 @@ private static string InitializeIndexFileAndSelectedFile(string fileContent, str
             string html = Microsoft.PowerToys.FilePreviewCommon.MonacoHelper.ReadIndexHtml();
 
             html = html.Replace("[[PT_LANG]]", vsCodeLangSet, StringComparison.InvariantCulture);
-            html = html.Replace("[[PT_WRAP]]", wrapText ? "1" : "0", StringComparison.InvariantCulture);
-            html = html.Replace("[[PT_CONTEXTMENU]]", "0", StringComparison.InvariantCulture);
-            html = html.Replace("[[PT_STICKY_SCROLL]]", stickyScroll ? "1" : "0", StringComparison.InvariantCulture);
+            html = html.Replace("[[PT_WRAP]]", wrapText ? "true" : "false", StringComparison.InvariantCulture);
+            html = html.Replace("[[PT_CONTEXTMENU]]", "false", StringComparison.InvariantCulture);
+            html = html.Replace("[[PT_STICKY_SCROLL]]", stickyScroll ? "true" : "false", StringComparison.InvariantCulture);
```

Comment:
Once again here, why the change of logic? I'm not saying it's wrong, but I'd like to understand the reasoning behind it.